docs/snippets/ov_hetero.cpp

```diff
@@ -43,14 +43,14 @@ compiled_model = core.compile_model(model, "HETERO", ov::device::priorities("GPU
 
 {
 //! [configure_fallback_devices]
-auto compiled_model = core.compile_model(model, "HETERO",
-    // GPU with fallback to CPU
-    ov::device::priorities("GPU", "CPU"),
-    // profiling is enabled only for GPU
-    ov::device::properties("GPU", ov::enable_profiling(true)),
-    // FP32 inference precision only for CPU
-    ov::device::properties("CPU", ov::hint::inference_precision(ov::element::f32))
-);
+    auto compiled_model = core.compile_model(model, "HETERO",
+        // GPU with fallback to CPU
+        ov::device::priorities("GPU", "CPU"),
```

Comment:
Why here do we have `ov::device::`? Should we remove device too?

Reply:
It is `device_priorities` property for aggregated plugin. Not any other priorities.